test/admission/validating-admission-policy/kubeconfig-org-a

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+clusters:
+- cluster:
+    insecure-skip-tls-verify: true
+    server: https://localhost:30443/apis/resourcemanager.miloapis.com/v1alpha1/organizations/test-vap-org-a/control-plane
+  name: org-test-vap-org-a
+contexts:
+- context:
+    cluster: org-test-vap-org-a
+    user: user-1001
+  name: org-test-vap-org-a
+current-context: org-test-vap-org-a
+kind: Config
+preferences: {}
+users:
+- name: user-1001
```

Comment:
Could change this to align with the name of the user instead of the previously used UID. Similar feedback for other places where 1001 is used. 

```suggestion
- name: user-admin
```